### DISCUSSION TITLE
Закрывается консоль

### DISCUSSION THREAD

**Discussion by IBETTERI** [atom/atom#22481] [Q&A]
Нажимаю на f5 что бы вызвать консоль, а она тут же закрывается. что делать?

**Comment by ThatXliner** [comment]
Что вы имеете в виду под консолью? Вы имеете в виду консоль JavaScript?

**Reply by IBETTERI** [reply]
Не консаль пайтон

**Reply by ThatXliner** [reply]
Вероятно, это проблема с пакетом, который вы используете для консоли Python (при условии, что вы имеете в виду оболочку)

**Reply by IBETTERI** [reply]
у пакета практически  1000000 скачиваний

**Reply by ThatXliner** [reply]
вы можете захотеть открыть проблему в репозитории пакетов

**Comment by kazaksaeed** [comment]
saeedkazak

در تاریخ چهارشنبه ۲۶ مهٔ ۲۰۲۱،‏ ۰:۴۷ ThatXliner ***@***.***>
نوشت:
